gradle-baseline-java/src/test/groovy/com/palantir/baseline/BaselineEclipseIntegrationTest.groovy

```diff
@@ -20,7 +20,7 @@ import com.google.common.base.Charsets
 import com.google.common.io.Files
 import nebula.test.IntegrationSpec
 import nebula.test.functional.ExecutionResult
-import org.apache.commons.io.FileUtils
+import org.gradle.internal.impldep.org.apache.commons.io.FileUtils
```

Comment:
They removed the dependency on `org.apache.commons.io.FileUtils` and added `org.gradle.internal.impldep.org.apache.commons.io.FileUtils` instead.